python_modules/libraries/dagster-embedded-elt/dagster_embedded_elt/dlt/translator.py

```diff
@@ -11,6 +11,8 @@
 
 @dataclass
 class DagsterDltTranslator:
+    metadata_by_resource_name: Optional[Mapping[str, Any]] = None
```

Comment:
Feels weird to have this in the base `DagsterDltTranslator` class. If a user wants to have this, you can subclass `DagsterDltTranslator` and add it yourself as a property. [1]